fmriprep/config.py

```diff
@@ -304,6 +304,9 @@ def init(cls):
             }
         })
 
+        if cls.omp_nthreads is None:
+            cls.omp_nthreads = min(cls.nprocs - 1 if cls.nprocs > 1 else os.cpu_count(), 8)
```

Comment:
```suggestion
            cls.omp_nthreads = min(os.cpu_count(), 8)
```
I've been willing to remove that nprocs - 1 for a long while. I think we overstepped on the responsibilities of the OS' scheduling. There's really no gain in having a bunch (if any) of fast, lightweight processes running in parallel to antsRegistration in one less cpu.

If you want to have one process running next to antsRegistration with 8 CPUs, it is much better to have --omp-nthreads 8 and make --nprocs 9 than removing one from ants.

Reply:
IMO the default behavior should be the best performance (on a typical dataset) without overcommitting processors, and the flags can be used to adapt. Have we tested whether leaving a core free to clear the small-job queue performs better or worse than `omp_nthreads = min(8, cpu_count())`?

If we haven't tested, let's make that a separate issue and just get this one in to get us back to the behavior users have come to expect.

Reply:
Sure, just to be clear - I'm not endorsing here to overcommit processors, I'm just saying that removing one available thread to processes does not make much sense.

And yet, we allow users to set that via the `--omp-settings` flag anyways.

That said, I'm fine to go ahead with this and do `omp_nthreads = min(8, cpu_count())` on a different PR. I haven't tested quantitatively, but in my experience, I'm sure it is much better. That `- 1` forces me to always set `--omp-nthreads`, which is a bit annoying.

Reply:
Opened #2078. Thanks for your forbearance.